tools/main.h

```diff
@@ -32,13 +32,36 @@
 #define MAIN_H
 
 #include <sapi/tpm20.h>
+#include <stdbool.h>
 #include "options.h"
 
+bool output_enabled;
```

Comment:
You're getting lucky here that this is a tentative declaration and the linker isn't complaining. This is extern by default, but I would explicitly extern this and put the definition in main.c. This way we can avoid any linker issues if the code gets moved around later.